store/subscription/subscription.go

```diff
@@ -29,6 +29,7 @@ type Store interface {
 	BatchInsert(subs []*model.Subscription) error
 	BatchUpdateBlockNumber(blockNumber int64, addrs [][]byte) error
 	Find(blockNumber int64) (result []*model.Subscription, err error)
+	// FindByAddresses find address of old subscriptions
```

Comment:
why do we need to restrict this? if ppl pass in the addresses they want to search, i don't see why we shouldn't return those new subscriptions too.

this change is independent of the fee fix. right?